lib/src/main/kotlin/at/bitfire/ical4android/AndroidCompatTimeZoneRegistry.kt

```diff
@@ -67,13 +69,13 @@ class AndroidCompatTimeZoneRegistry(
            but most Android devices don't now Europe/Kyiv yet.
            */
         if (tz.id != androidTzId) {
-            logger.warning("Using Android TZID $androidTzId instead of ical4j ${tz.id}")
+            logger.info("Using ical4j timezone ${tz.id} data to construct Android timezone $androidTzId")
```

Comment:
I have indeed meant `logger.fine` ;)

Reason: INFO is for about what is currently being done in an overview scope. For instance "syncing collection", "downloading task", "job is cancelled". Basically everything that is interesting to understand what DAVx5 currently does.

Debug level (FINE) is for details about _how_ current operations are done, which are rather interesting when debugging something.

The fact that a timezone definition is used instead of another one is only important for debugging things with timezones, so I'd use a debug level (FINE) for that.

Reply:
> I indeed meant `logger.fine` ;)

oh .. my bad

Reply:
No problem, I'll set it back